### PR TITLE
[CARBONDATA-3065]Correct the error message for inverted index columns and update the docment

### PR DESCRIPTION
**Pull request by brijoobopanna**
Changes in this PR:
1. Modified the error message when column present in Inverted index is not present in Schema
2. Updated the table property in user document

Be sure to do all of the following checklist to help us incorporate 
your contribution quickly and easily:

 - [NA] Any interfaces changed?
 
 - [NA] Any backward compatibility impacted?
 
 - [Yes] Document update required?

 - [Yes] Testing done through UT
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - How it is tested? Please attach test report.
        - Is it a performance related change? Please attach the performance test report.
        - Any additional information to help reviewers in testing this change.
       
 - [NA] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA.

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/1414/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/1624/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.1, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/9672/

File: integration/spark-datasource/src/test/scala/org/apache/spark/sql/carbondata/datasource/SparkCarbonDataSourceTest.scala [hunk 1]
```diff
@@ -1345,6 +1345,15 @@ class SparkCarbonDataSourceTest extends FunSuite with BeforeAndAfterAll {
     spark.sql("drop table if exists fileformat_drop_hive")
   }
 
+  test("validate the columns not present in schema") {
+    spark.sql("drop table if exists validate")
```
Comment (kunal642) [inline integration/spark-datasource/src/test/scala/org/apache/spark/sql/carbondata/datasource/SparkCarbonDataSourceTest.scala]: Need to drop table after test case completion. better to add in afterAll() too

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/1444/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.1, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/9702/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/1654/

**Comment by kunal642** [comment]
LGTM

**Pull request closed without merging**